### PR TITLE
Teacher pane ui

### PR DESCRIPTION
**Pull request by Bjvanminnen**
image

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
Been sending lots of PRs Brad's way, so going to try to send one elsewhere :) Changes are mostly style related, and thus probably not super interesting.

File: apps/src/code-studio/components/progress/StageLock.jsx [hunk 1]
```diff
@@ -10,18 +10,23 @@ import { openLockDialog, closeLockDialog, lockStage } from '../../stageLockRedux
...
+    maxWidth: 240,
+    overflow: 'hidden'
+  },
   lockSettingsText: {
     marginLeft: 10
   },
   warning: {
-    position: 'relative',
-    top: 2
+    marginBottom: 5,
```
Comment (caleybrock) [inline apps/src/code-studio/components/progress/StageLock.jsx:22]: to prove I actually read all of this, my only nit is that there are there are a couple of extra commas in the styles objects

**Comment by caleybrock** [comment]
lgtm

**Comment by caleybrock** [comment]
also this is looking much better :)

**Pull request merged**